logstash-core/src/main/java/org/logstash/execution/AbstractPipelineExt.java

```diff
@@ -521,23 +525,8 @@ public final IRubyObject initializeFlowMetrics(final ThreadContext context) {
         this.flowMetrics.add(concurrencyFlow);
         storeMetric(context, flowNamespace, concurrencyFlow);
 
-        // collect the queue_persisted_growth_events & queue_persisted_growth_bytes metrics if only persisted queue is enabled.
-        if (getSetting(context, QueueFactoryExt.CONTEXT_NAME).asJavaString()
-                .equals(QueueFactoryExt.PERSISTED_TYPE)) {
-
-            final RubySymbol[] queueNamespace = buildNamespace(QUEUE_KEY);
-            final RubySymbol[] queueCapacityNamespace = buildNamespace(QUEUE_KEY, CAPACITY_KEY);
-
-            final Supplier<NumberGauge> eventsGaugeMetricSupplier = () -> initOrGetNumberGaugeMetric(context, queueNamespace, EVENTS_KEY).orElse(null);
-            final FlowMetric growthEventsFlow = createFlowMetric(QUEUE_PERSISTED_GROWTH_EVENTS_KEY, eventsGaugeMetricSupplier, () -> uptimeInPreciseSeconds);
-            this.flowMetrics.add(growthEventsFlow);
-            storeMetric(context, flowNamespace, growthEventsFlow);
-
-            final Supplier<NumberGauge> queueSizeInBytesMetricSupplier = () -> initOrGetNumberGaugeMetric(context, queueCapacityNamespace, QUEUE_SIZE_IN_BYTES_KEY).orElse(null);
-            final FlowMetric growthBytesFlow = createFlowMetric(QUEUE_PERSISTED_GROWTH_BYTES_KEY, queueSizeInBytesMetricSupplier, () -> uptimeInPreciseSeconds);
-            this.flowMetrics.add(growthBytesFlow);
-            storeMetric(context, flowNamespace, growthBytesFlow);
-        }
+        initializePqFlowMetrics(context, flowNamespace, uptimeInPreciseSeconds);
+        initializePluginFlowMetrics(context, uptimeInPreciseSeconds);
```

Comment:
BUG: `initializePluginFlowMetrics`'s 2nd argument is called `uptimeInPreciseMillis`, but we are giving it a seconds-level view. To prevent this type of issue from happening, we should pass our actual `UptimeMetric` through and let it create the scaled view it needs.

```suggestion
        initializePqFlowMetrics(context, flowNamespace, uptimeMetric);
        initializePluginFlowMetrics(context, uptimeMetric);
```